doc/src/fix_bond_react.rst

```diff
@@ -311,8 +311,8 @@ either 'none' or 'charges.' Further details are provided in the
 discussion of the 'update_edges' keyword. The fifth optional section
 begins with the keyword 'Constraints' and lists additional criteria
 that must be satisfied in order for the reaction to occur. Currently,
-there are four types of constraints available, as discussed below:
-'distance', 'angle', 'dihedral', and 'arrhenius'.
+there are five types of constraints available, as discussed below:
+'distance', 'angle', 'dihedral', 'arrhenius', and 'RMSD'.
```

Comment:
wouldn't it be more consistent to have the 'RMSD' keyword lowercase like all the others (and like in most of LAMMPS' commands)?